.gitignore

```diff
@@ -184,3 +184,6 @@ knip.json
 type-docs
 docs.json
 tsconfig.typedoc.tmp.json
+
+# Storybook
+storybook-static/
```

Comment:
Could we call this one `dist-storybook`? we're using that pattern for other output dirs

Reply:
Done. It is now `dist-storybook`